aggregators/rules/ipg.js

```diff
@@ -1,17 +1,49 @@
+const jsDOM = require("jsdom");
 class IPG {
     constructor() {
-        this.Location = "http://blogs.magicjudges.org/rules/ipg/";
+        this.location = "http://blogs.magicjudges.org/rules/ipg";
     }
 
     find(parameter, callback) {
-        //todo
+        jsDOM.env(
+            this.location + parameter,
+            ["http://code.jquery.com/jquery.js"],
```

Comment:
Why not have a local jquery.js and load that once, instead of requesting it every time?